src/main/java/the_warlord/cards/warlord/Improvise.java

```diff
@@ -0,0 +1,35 @@
+package the_warlord.cards.warlord;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import the_warlord.WarlordMod;
+import the_warlord.cards.CustomWarlordModCard;
+import the_warlord.characters.Warlord;
+import the_warlord.powers.ImprovisePower;
+
+public class Improvise extends CustomWarlordModCard {
+    public static final String ID = WarlordMod.makeID(Improvise.class);
+
+    private static final CardRarity RARITY = CardRarity.UNCOMMON;
+    private static final CardTarget TARGET = CardTarget.SELF;
+    private static final CardType TYPE = CardType.SKILL;
+    public static final CardColor COLOR = Warlord.Enums.WARLORD_CARD_COLOR;
+    private static final int COST = 3;
+    private static final int UPGRADED_COST = 2;
+    public Improvise() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+    }
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        addToBot(new ApplyPowerAction(p, p, new ImprovisePower(p, 2)));
```

Comment:
The amount for this power should be 1 (and preferably also come from a static variable).